src/vs/workbench/parts/extensions/electron-browser/extensionEditor.ts

```diff
@@ -369,6 +392,22 @@ export class ExtensionEditor extends BaseEditor {
 		this.extensionActionBar.push([disabledStatusAction, reloadAction, updateAction, enableAction, disableAction, installAction, maliciousStatusAction], { icon: true, label: true });
 		this.transientDisposables.push(enableAction, updateAction, reloadAction, disableAction, installAction, maliciousStatusAction, disabledStatusAction);
 
+		const ignoreAction = this.instantiationService.createInstance(IgnoreAction);
+		ignoreAction.extension = extension;
+		ignoreAction.onIgnored = () => {
+			const activeViewlet = this.viewletService.getActiveViewlet();
+			if (activeViewlet instanceof ExtensionsViewlet) {
+				(<ExtensionsViewlet>activeViewlet).refreshRecommendedExtensions();
+			}
+			addClass(this.header, 'ignored');
```

Comment:
Do we really need to add this class?
From what I can see in the css file, its used in the same situation where the `recommended` label is used.